src/coffee/mappings.coffee

```diff
@@ -88,11 +97,6 @@ class CustomFieldMappings
       @errors.push "[row #{rowIndex}:#{typeDefinitionKey}] Can not parse money '#{rawMoney}'!"
       return
 
-    validCurr = CONS.REGEX_CUR.exec matchedMoney[1]
```

Comment:
This is a needed check, also wanted [here](https://github.com/sphereio/sphere-node-product-csv-sync/blob/master/src/coffee/mapping.coffee#L302). Any reason for removing it?